### PR TITLE
[NO-JIRA] - adding Dockerfile to app-sre build process

### PR DESCRIPTION
**Pull request by msmagnanijr**
Signed-off-by: Mauricio Magnani <msmagnanijr@gmail.com>

[Insert JIRA Issue number and update link](https://issues.redhat.com/browse/MGDOBR-)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

**Comment by msmagnanijr** [comment]
@all Some points:

- I'm using "-Dquickly " because there is some test failing so to proceed at this point I added this. We can investigate and fix this later.

- I'm using "-Quarkus.container-image.build=false" to prevent the Fleet shard image from being built, as it would need the Docker binary and we don't have it in ubi8/ubi-minimal

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5-240 as builder
```
Comment (r00ta) [inline docker/Dockerfile:2]: what about renaming this file `Dockerfile.appsre` or something similar?

Comment (msmagnanijr) [reply docker/Dockerfile:2]: @r00ta sure, it makes sense to make the objective clear.

File: docker/Dockerfile.appsre [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+COPY --from=builder --chown=1001 /workspace/manager/target/quarkus-app/lib/ /deployments/lib/
+COPY --from=builder --chown=1001 /workspace/manager/target/quarkus-app/*.jar /deployments/
+COPY --from=builder --chown=1001 /workspace/manager/target/quarkus-app/app/ /deployments/app/
+COPY --from=builder --chown=1001 /workspace/manager/target/quarkus-app/quarkus/ /deployments/quarkus/
+
+EXPOSE 8080
+USER 1001
+
+ENTRYPOINT [ "/deployments/run-java.sh" ]
```
Comment (danielezonca) [inline docker/Dockerfile.appsre:49]: Can you please refactor starting from this https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/docker/Dockerfile.jvm#L78-L93 
(of course making sure to preserve the `--from=builder` etc)?

It seems to be the "latest" version of Quarkus Dockerfile

**Review by danielezonca — APPROVED** [review]
LGTM 👍

**Review by r00ta — APPROVED** [review]

File: app-interface/buid_project.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```
Comment (manstis) [inline app-interface/buid_project.sh:1]: IDK if you want to rename this file to `bui*l*d_project.sh`?

**Pull request merged**